SU-SWS/decanter
# Split input.scss into several files and add new input-base placeholder

# READY FOR REVIEW

# Summary
- The current input.scss file inside the elements directory has styling for many input elements that might not be used for every project (eg, range, custom radio button/check boxes styling). This PR splits it up into several files so developers can choose to cherry pick the elements they need. 
- Added input-base placeholder.
- Removed "role=navigation" for Global Footer nav (Level Access recommendation).
- Note: styling for radio button/checkbox/range probably need work later, but the purpose of this PR isn't to change them so no work has been done to them.

# Needed By (Date)
- Whenever

# Urgency
- N/A

# Steps to Test

1. Pull this branch and compile styleguide
2. Check that everything that has input elements still functions the same.

# Affected Projects or Products
- Decanter
- Redwood WP theme

# Associated Issues and/or People
- #475 
- @JBCSU
